### ISSUE TITLE
The unit of the velocity

### ISSUE THREAD

**Issue by X-chaser** [carla-simulator/imitation-learning#92]
The annotation in the file agents/imitation_learning.py says that the maximum speed in training set is 90km/h. So I analyzed the speed label in training data, and the result shows that the values of the speed labels do have a limitation of 90. But the point that puzzels me is when regularizing the speed, the code divides the speed by 25, which I think means 25m/s. Isn't it should divide the speed by 90? And this results my question: what is the unit of the speed label? Thanks for your answer!
